core/bin/genesis_generator/src/main.rs

```diff
@@ -110,7 +111,7 @@ pub(crate) fn encode_yaml<T: ReflectMessage>(x: &T) -> anyhow::Result<String> {
     let mut serializer = Serializer::new(vec![]);
     let opts = prost_reflect::SerializeOptions::new()
         .use_proto_field_name(true)
-        .stringify_64_bit_integers(true);
+        .stringify_64_bit_integers(false);
```

Comment:
What does this do btw?

Reply:
It changes the representation from strings to numbers. otherwise it can't be parsed properly